src/cc/bcc_proc.c

```diff
@@ -82,6 +83,43 @@ int bcc_mapping_is_file_backed(const char *mapname) {
     STARTS_WITH(mapname, "[vsyscall]"));
 }
 
+/*
+Finds a file descriptor for a given inode if it's a memory-backed fd.
+*/
+char *bcc_procutils_memfd_path(int pid, uint64_t inum) {
+  char *path = NULL;
+  char *globstr;
+  char glob_buffer[1024];  // a ridiculous buffer size
+  int glob_size, i;
+  glob_t globbuf;
+  struct stat sb;
+
+  snprintf(glob_buffer, 1024, "/proc/%d/fd/*", pid);
+  glob_size = strlen(glob_buffer);
+
+  globstr = malloc(sizeof(char) * (glob_size + 1));
+  strcpy(globstr, glob_buffer);
+
+  glob(globstr, GLOB_NOSORT, NULL, &globbuf);
+  free(globstr);
+
+  for ((i = 0 && path == NULL); i < globbuf.gl_pathc; i++) {
+    char *pid_fd_path = globbuf.gl_pathv[i];
+    if (stat(pid_fd_path, &sb) == -1) {
+      printf("Error calling lstat on %s\n", pid_fd_path);
+    }
+
+    printf("inode: %ld %ld\n", sb.st_ino, inum);
```

Comment:
For error we should output to stderr and this informative info shouldn’t be printed by default

Reply:
Yeah i've already removed this, accidentally pushed it to the draft PR